docs/designers-developers/developers/data/data-core-block-editor.md

```diff
@@ -335,6 +335,18 @@ _Returns_
 
 -   `Array<Editor.InserterItem>`: Items that appear in inserter.
 
+<a name="getKeyboardMode" href="#getKeyboardMode">#</a> **getKeyboardMode**
```

Comment:
What options do we foresee for keyboard modes? If there's only two, and in interest to avoid [magic strings](https://en.wikipedia.org/wiki/Magic_number_(programming)#Unnamed_numerical_constants), maybe we should repurpose these to return booleans reflecting whether the mode is specifically one or the other?

e.g. `isNavigationMode` or `isEditMode` ?

**Edit:** I see now this was [discussed previously](https://github.com/WordPress/gutenberg/pull/16500/files#r304848958), but I remain skeptical. What other modes do you foresee?

Reply:
What if we want something to navigate the aria-regions directly in the future... I mean, to be honest I don't care that much. You both seem to think a boolean should be enough, I'll update :)